modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/util/BSTValidator.java

```diff
@@ -88,8 +88,9 @@ public Credential validate(Credential credential, RequestData requestData) throw
             log.error(msg, e);
             throw new WSSecurityException(msg, e);
         } catch (WindowsDeviceEnrolmentException e) {
-            String msg = "Error occurred in validating request token.";
-            log.error(msg);
+            String msg = "Authentication Failure occurred due to binary security token.";
+            log.error(msg, e);
```

Comment:
"Authentication Failure...., here Failure could be "failure", no need to make it start with a capital.
Also error message is bit unclear as it just mentions "failure due to binary security token"